process_qsos.m

```diff
@@ -401,9 +400,10 @@
         dla_omega2 = this_omega2 .* absorption.^2;
         
         % Add a penalty for short spectra: the expected reduced chi^2 of each spectral pixel that would have been observed.
+        lambda_observed = (max_observed_lambda - min_observed_lambda);
```

Comment:
I think this line should before `this_sample_log_likelihoods_no_dla `?

Reply:
Oops, good point!